app/Http/Controllers/BudgetGeneralController.php

```diff
@@ -47,7 +46,7 @@ public function update(
                 $budgetColumnSetting,
                 $updateBudgetColumnSettingRequest
             );
-        } catch (\Throwable $t) {
+        } catch (Throwable $t) {
             Log::error('Not able to save budget general setting for reason: ' . $t->getMessage());
```

Comment:
In this case changes are not made because of boyscouting rule - i'll keep report in mind and add it later: Analysed that desired interface to use in constructor is: Illuminate\Contracts\Debug\ExceptionHandler